src/components/tiptap/bubbleMenu/DropdownBubbleMenu.tsx

```diff
@@ -1,19 +1,51 @@
 import { MenuItem, Select, SelectChangeEvent } from '@mui/material'
-import { FC, ReactNode, useState } from 'react'
+import { FC, ReactNode, useEffect, useState } from 'react'
 
-import { AutofillIcon, H1Icon, H2Icon, H3Icon, TextIcon } from '@/icons'
+import {
+  CalloutIcon,
+  H1Icon,
+  H2Icon,
+  H3Icon,
+  LinkIcon,
+  TextIcon,
+} from '@/icons'
 import { TiptapEditorUtils } from '@/utils/tiptapEditorUtils'
 import { Editor } from '@tiptap/react'
+import { useAppState } from '@/hooks/useAppState'
 
 interface IBubbleMenuContainer {
   editor: Editor
 }
 
 const DropdownBubbleMenu: FC<IBubbleMenuContainer> = ({ editor }) => {
-  const [selectedFormatter, setSelectedFormatter] = useState('')
+  const [selectedFormatter, setSelectedFormatter] = useState('Text')
 
   const tiptapEditorUtils = new TiptapEditorUtils(editor)
 
+  const appState = useAppState()
+
+  useEffect(() => {
+    const parent = editor.state.selection.$anchor.parent
+    const level = parent.attrs.level
+    const name = parent.type.name
+
+    if (name === 'heading' && level === 1) {
+      setSelectedFormatter('Heading 1')
```

Comment:
@aatbip wouldn't it be better to create an enum or constant for possible formatters?